modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -58,12 +58,21 @@ public class UpgradeCatchAllCheck {
 			companyId, 551, "test", start, end);
 	}
 
+	public void method(HttpServletRequest httpServletRequest) {
+	   _commerceAccountHelper.getCurrentAccountEntry(0, httpServletRequest);
+
+	   _commerceAccountHelper.getCurrentCommerceAccount(null);
+	}
+
 	@Reference
 	private AssetVocabularyLocalService _assetVocabularyLocalService;
 
 	@Reference
 	private AssetVocabularyService _assetVocabularyService;
 
+	@Reference
+    private CommerceAccountHelper _commerceAccountHelper;
+
```

Comment:
@alberttgomes I believe that we don't need this extra line after this new reference, can we remove it?

Reply:
maybe, this will fix after you get the new versions of the required commits.